File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w = 0
    h = 0
    for size in sizes:
        w_temp = max(size)
        h_temp = min(size)
        w = max(w_temp, w)
        h = max(h_temp, h)
    answer = w * h
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.swift
import Foundation

func solution(_ sizes:[[Int]]) -> Int {
    var w = 0
    var h = 0
    sizes.forEach {
        if let wTemp = $0.max(),
           let hTemp = $0.min(){
            w = max(wTemp, w)
            h = max(hTemp, h)
        }
    }
    return w * h
}